src/engine/SortPerformanceEstimator.cpp

```diff
@@ -101,45 +103,25 @@ SortPerformanceEstimator::SortPerformanceEstimator(
   LOG(INFO) << "Done creating sort estimates" << std::endl;
 }
 
-double SortPerformanceEstimator::EstimateSortTimeInSeconds(
+double SortPerformanceEstimator::estimateSortTimeInSeconds(
     size_t numRows, size_t numCols) const noexcept {
   // Return the index of the element in the !sorted! `sampleVector`, which is
   // closest to 'value'
   auto getClosestIndex = [](const auto& sampleVector, size_t value) -> size_t {
-    // Get iterator and index of first element in the vector that is >= value
-    auto firstGreaterEqualIterator =
-        std::find_if(sampleVector.begin(), sampleVector.end(),
-                     [value](const auto& element) { return element >= value; });
-    auto firstGreaterEqualIndex =
-        firstGreaterEqualIterator - sampleVector.begin();
-
-    if (firstGreaterEqualIterator == sampleVector.end()) {
-      // Even the last/greatest element in the vector is smaller than `value`,
-      // return the index of the last element.
-      return firstGreaterEqualIndex - 1;
-    }
-
-    if (firstGreaterEqualIterator == sampleVector.begin()) {
-      // The first/smallest element in the vector is already >= `value`, return
-      // the index of the first element.
-      return 0;
-    }
-
-    // Value lies somewhere between *(firstGreaterEqualIterator - 1) and
-    // *firstGreaterEqualIterator. Find out which is closer and return the
-    // appropriate index.
-    auto distanceToCurrent =
-        std::abs(static_cast<long long>(*firstGreaterEqualIterator) -
-                 static_cast<long long>(value));
-    auto distanceToPrevious =
-        std::abs(static_cast<long long>(*(firstGreaterEqualIterator - 1)) -
-                 static_cast<long long>(value));
-
-    if (distanceToCurrent <= distanceToPrevious) {
-      return firstGreaterEqualIndex;
-    } else {
-      return firstGreaterEqualIndex - 1;
-    }
+    // Return the absolute distance to `value`
```

Comment:
A remark: It's good to quote references to variable names in comment. Is there a standard for this? I have always used "..." in the past. I know that Markdown uses `...` but I am not sure how relevant markdown is for comments in code. Anyway, the most important thing is that we use it consistently